_includes/pbc/all/install-features/202404.0/marketplace/install-the-marketplace-merchant-feature.md

```diff
@@ -29,37 +29,14 @@ Make sure the following modules have been installed:
 | MerchantProfile           | vendor/spryker/merchant-profile             |
 | MerchantProfileDataImport | vendor/spryker/merchant-profile-data-import |
 | MerchantProfileGui        | vendor/spryker/merchant-profile-gui         |
-| MerchantSearch            | vendor/spryker/merchant-search              |
-| MerchantSearchExtension   | vendor/spryker/merchant-search-extension    |
 | MerchantUser              | vendor/spryker/merchant-user                |
 | MerchantUserGui           | vendor/spryker/merchant-user-gui            |
```

Comment:
* We have to find the "MerchantStorage" word in the document and clean up those parts.